serve/src/main.py

```diff
@@ -110,33 +110,62 @@ def load_on_device(
         self,
         model_dir,
         device: Literal["cpu", "cuda", "cuda:0", "cuda:1", "cuda:2", "cuda:3"] = "cpu",
+        started_via_api: bool = False,
+        deploy_params: Dict[str, Any] = None,
     ):
```

Comment:
def load_model(checkpoint, therethold, config, arg1, arg2, ....):
     """ docstring
    """
     implementation
     pass

def deploy():
     # get_segging_from_ui
     x  = gui.get_value()
     y = GUI. get_value()
     
     # load
     load_model(x, y)
     ....


in SDK


@server.post("/get_deploy_settings")
def _get_deploy_settings()
     return self.load_model.__arguments + self.load_model.__docstring

@server.post("/deploy")
def _get_deploy_settings(self, state)
    if self.ise_deploed:
          self.stop()
    self.load_model(** state["deploy_settings])

============================================================

def load_on_device(
        self,
        model_dir,
        device: Literal["cpu", "cuda", "cuda:0", "cuda:1", "cuda:2", "cuda:3"] = "cpu",
        deploy_params: Dict[str, Any] = None,
    ):
....

def get_deploy_params()
    return {"task": ..., "dfsdf", ...}

@server.post(/deploy)
def _deploy(d, state):
   doanwlod(state["files"]) -> model_dir
   serve()





client 
api.app.start(autostart=False)
d = api.app.nn.get_deploy_params()
d[x] = 777
api.app.nn.deploy(d)

Reply:
```py
def define_gui()
      tab= sly.widget.RadioTab()
      self.layout_tab1 = ...
      self.layout_tab2 = ...
      self.tab= sly.widget.RadioTab(layout_tab1, layout_tab2)
      return layout
```